tests/python/integration/test_tuning.py

```diff
@@ -174,7 +173,14 @@ def runner(target, dev):
 
         assert len(results) == 20
 
-        successful_results = [r for r in results if r.error_no == autotvm.MeasureErrorNo.NO_ERROR]
+        successful_results = [
+            r
+            for r in results
+            if r.error_no == autotvm.MeasureErrorNo.NO_ERROR
+            # Autotvm can filter some records before building if we know they won't work ahead of time.
+            # We can't guarantee we sample at least one good record so we count these as success too
+            or r.error_no == autotvm.MeasureErrorNo.INSTANTIATION_ERROR
```

Comment:
no idea what I'm talking about here but wouldn't this just hide the flakiness the same as the `xfail` (i.e. if this line happens the test is bogus)? Is there a way to make it deterministic on a known good rng seed?

Reply:
Nah it's expected to sometimes fail during the tuning process so we would expect each result to have no error or an instantiation error (which indicates we caught it and didn't build). I don't know enough about how to make things deterministic, perhaps by replacing the workload with something simpler where it can never fail.